### ISSUE TITLE
Remnant python2.7 ambiguity in Makefile to be removed

### ISSUE THREAD

**Issue by aoeftiger** [PyCOMPLETE/PyHEADTAIL#124]
When running `make` in order to compile the `errfff` complex error function (from the CERN library), we should resort to `f2py3` instead of `f2py` as on some systems `f2py` still falls back to a present python2.7 installation.

**Comment by aoeftiger** [comment]
(One would typically find an error message along the lines of 
```
creating /tmp/tmp6fgkWA/tmp
creating /tmp/tmp6fgkWA/tmp/tmp6fgkWA
creating /tmp/tmp6fgkWA/tmp/tmp6fgkWA/src.linux-x86_64-2.7
compile options: '-I/tmp/tmp6fgkWA/src.linux-x86_64-2.7 -I/home/oeftiger/.local/lib/python2.7/site-packages/numpy/core/include -I/usr/include/python2.7 -c'
x86_64-linux-gnu-gcc: /tmp/tmp6fgkWA/src.linux-x86_64-2.7/fortranobject.c
x86_64-linux-gnu-gcc: /tmp/tmp6fgkWA/src.linux-x86_64-2.7/errfffmodule.c
/tmp/tmp6fgkWA/src.linux-x86_64-2.7/errfffmodule.c:14:10: fatal error: Python.h: No such file or directory
   14 | #include "Python.h"
      |          ^~~~~~~~~~
compilation terminated.
In file included from /tmp/tmp6fgkWA/src.linux-x86_64-2.7/fortranobject.c:2:
/tmp/tmp6fgkWA/src.linux-x86_64-2.7/fortranobject.h:7:10: fatal error: Python.h: No such file or directory
    7 | #include "Python.h"
      |          ^~~~~~~~~~
compilation terminated.
error: Command "x86_64-linux-gnu-gcc -pthread -fno-strict-aliasing -Wdate-time -D_FORTIFY_SOURCE=2 -g -fdebug-prefix-map=/build/python2.7-1x6jhf/python2.7-2.7.18~rc1=. -fstack-protector-strong -Wformat -Werror=format-security -fPIC -I/tmp/tmp6fgkWA/src.linux-x86_64-2.7 -I/home/oeftiger/.local/lib/python2.7/site-packages/numpy/core/include -I/usr/include/python2.7 -c /tmp/tmp6fgkWA/src.linux-x86_64-2.7/errfffmodule.c -o /tmp/tmp6fgkWA/tmp/tmp6fgkWA/src.linux-x86_64-2.7/errfffmodule.o -MMD -MF /tmp/tmp6fgkWA/tmp/tmp6fgkWA/src.linux-x86_64-2.7/errfffmodule.o.d" failed with exit status 1
make: *** [Makefile:20: errfff] Error 1
```
-- note the `python2.7` in the paths, although `anaconda3` has been set up with its standard pre-pending path approach, 
```
# >>> conda initialize >>>
# !! Contents within this block are managed by 'conda init' !!
__conda_setup="$('/home/oeftiger/anaconda3/bin/conda' 'shell.bash' 'hook' 2> /dev/null>
if [ $? -eq 0 ]; then
    eval "$__conda_setup"
else
    if [ -f "/home/oeftiger/anaconda3/etc/profile.d/conda.sh" ]; then
        . "/home/oeftiger/anaconda3/etc/profile.d/conda.sh"
    else
        export PATH="/home/oeftiger/anaconda3/bin:$PATH"
    fi
fi
unset __conda_setup
# <<< conda initialize <<<
```
)

**Comment by aoeftiger** [comment]
Fixed with PR #125
